packages/entity-browser/src/modules/entityBrowser/sagas.js

```diff
@@ -16,7 +16,7 @@ export default function* sagas() {
 
 export function* initialize() {
   const entityBrowser = yield select(entityBrowserSelector)
-  let {entityName, formBase} = entityBrowser
+  let {entityName, formBase, showSearchForm, disableSimpleSearch, simpleSearchFields} = entityBrowser
```

Comment:
maybe we should set this const and add seperate var for formBase. Seams ugly that we use let for all there vars just because of one